src/Illuminate/Container/Container.php

```diff
@@ -1024,6 +1026,26 @@ protected function resolvePrimitive(ReflectionParameter $parameter)
             return [];
         }
 
+        if (isset($this['config'])) {
+            foreach ($parameter->getAttributes() as $attribute) {
```

Comment:
```suggestion
            foreach ($parameter->getAttributes(InjectedConfig::class) as $attribute) {
```

Retrieving all attributes without specifying a name is dangerous and not very performant as we get all attributes specified even ones that can not be instantiated as they are from dev dependencies, causing errors for the users application.